assets/css/style.css.liquid

```diff
@@ -775,6 +778,14 @@ table {
   background-color: var(--darkturquoise);
 }
 
+.button--hero--blue {
+  background-color: #1292e8;
```

Comment:
Maybe this should be defined as variables like the rest of colors... so they'll be simpler to switch for a dark theme in the future.

Reply:
Yep, let's switch it to also be .button--hero and I will expand the color palette with more hues/shades/tints.